web/modules/custom/dbcdk_community/src/CommunityApiConfiguration.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\dbcdk_community\CommunityApiConfiguration.
+ */
+
+namespace Drupal\dbcdk_community;
+
+use DBCDK\CommunityServices\Configuration as Configuration;
+
+/**
+ * Community API Configuration Class.
+ *
+ * This class contains the configuration needed for the ApiClient to
+ * communicate with the Community Service.
+ *
+ * @See DBCDK\CommunityServices\ApiClient
+ */
+class CommunityApiConfiguration extends Configuration {
+
+  /**
+   * Set configuration for the Community Service Api Client.
+   */
+  public function __construct() {
+    parent::__construct();
+
+    $config = \Drupal::config('dbcdk_community.settings');
+    // The Community Service prefixes each endpoint with a slash,
+    // so we make sure the host does not end with one.
+    $host = rtrim($config->get('community_service_url'), '/');
+    $this->setHost($host);
```

Comment:
We should consider also adding an option for setting [the debug configuration](https://github.com/DBCDK/biblo-admin/blob/6ac5aa10f1b797b6529a4c65306792e9a178e289/lib/client/src/Configuration.php#L391-L391). This is not necessarily needed right now but could be useful going forward.